kythe/web/site/examples.md

```diff
@@ -93,7 +94,7 @@ then be stored in a [GraphStore]({{site.baseurl}}/docs/kythe-storage.html).
 java -Xbootclasspath/p:third_party/javac/javac*.jar \
   com.google.devtools.kythe.analyzers.java.JavaIndexer \
   $PWD/.kythe_compilations/java/kythe_java_com_google_devtools_kythe_analyzers_java_analyzer.java.kzip > entries
-# NOTE: https://kythe.io/phabricator/T40 -- the Java indexer should not be run in
+# NOTE: https://github.com/kythe/kythe/issues/491 -- the Java indexer should not be run in
```

Comment:
This should be https://github.com/kythe/kythe/issues/818, but it should also be fixed.